src/main/java/com/boot/bankservice/service/impl/AccountServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package com.boot.bankservice.service.impl;
+
+import com.boot.bankservice.exception.DataProcessingException;
+import com.boot.bankservice.model.Account;
+import com.boot.bankservice.repository.AccountRepository;
+import com.boot.bankservice.service.AccountService;
+import java.util.Optional;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class AccountServiceImpl implements AccountService {
+    private AccountRepository accountRepository;
+
+    @Override
+    public Account add(Account account) {
+        return accountRepository.save(account);
+    }
+
+    @Override
+    public Optional<Account> get(Long id) {
+        return accountRepository.findById(id);
+    }
+
+    @Override
+    public Account update(Account account) {
```

Comment:
what is the purpose in having 2 methods with the same realisation?)
I would agree that we should better have only 1 `save()` method

Reply:
when you want use save, you can use - save(), but when you want update, you can use - update(), after few month when I will forget that method save() do same thing, like update and save it's will be useful